pkg/auth/service.go

```diff
@@ -1878,25 +1919,32 @@ func (a *APIAuthService) ClaimTokenIDOnce(ctx context.Context, tokenID string, e
 	return a.validateResponse(res, http.StatusCreated)
 }
 
-func NewAPIAuthService(apiEndpoint, token string, secretStore crypt.SecretStore, cacheConf params.ServiceCache, timeout *time.Duration, emailer *email.Emailer) (*APIAuthService, error) {
+func NewAPIAuthService(apiEndpoint, token string, secretStore crypt.SecretStore, cacheConf params.ServiceCache, emailer *email.Emailer, logger logging.Logger) (*APIAuthService, error) {
+	if token == "" {
```

Comment:
Add INFO log? (something in the form of)
```go
logger.Info("no token provided for auth service, will use generated JWT token")
```

Reply:
wanted to capture the token ID so inside the generate func we have

```
	logging.Default().WithField("id", id).Info("generated auth api token")
```